src/amazon/AmazonProduct.js

```diff
@@ -58,7 +58,8 @@ class AmazonProduct {
         width: dimensions['ns2:Width'] ? dimensions['ns2:Width'] : 'UNKNOWN',
         height: dimensions['ns2:Height'] ? dimensions['ns2:Height'] : 'UNKNOWN',
         length: dimensions['ns2:Length'] ? dimensions['ns2:Length'] : 'UNKNOWN',
-        weight: dimensions['ns2:Weight'] ? dimensions['ns2:Weight'] : 'UNKNOWN'
+        weight: dimensions['ns2:Weight'] ? dimensions['ns2:Weight'] : 'UNKNOWN',
+        weightComputed: dimensions['ns2:Weight'] ? false : true
```

Comment:
I don't think I understand the purpose of this attribute. Is it even being used?

Reply:
It is not being used right now. It is assigned true for a product that initially had no weight returned from Amazon but for which we computed a weight using representative weight and false for a product that returned a weight from amazon. 
Whenever we are making a decision to buy such product, we can manually check its weight via Google search compared to what our representative value was and see if our ROI numbers make sense for shipping cost.